Runtime/Managers/EditMediator.cs

```
@@ -233,7 +233,13 @@ public async Task ExcecuteWebMessage(string type, JToken payload)
                     SendModelInfoMessage();
                     break;
                 case WebMessageType.changeObjectsMaterial:
-                    await _changeMaterialController.ChangeObjectMaterial((JObject)payload);
+                    TaskResult result = await _changeMaterialController.ChangeObjectMaterial((JObject)payload);
+                    Debug.Log(result.IsSuccess);
```

If I'm logging anything here, it would be `result.error` since we're actually swallowing that value.